i18n/de.json

```diff
@@ -454,10 +454,10 @@
     "settings.smtp.name": "SMTP",
     "settings.smtp.retries": "Wiederholungen",
     "settings.smtp.retriesHelp": "Maximale Anzahl an Wiederholungen, wenn eine Machricht fehlschlägt.",
-    "settings.smtp.sendTest": "Send e-mail",
+    "settings.smtp.sendTest": "E-mail senden",
     "settings.smtp.setCustomHeaders": "Benutzerdefinierten Header verwenden",
-    "settings.smtp.testConnection": "Test connection",
-    "settings.smtp.toEmail": "To e-mail",
+    "settings.smtp.testConnection": "Verbindung testen",
+    "settings.smtp.toEmail": "An E-mail",
```

Comment:
Isn't this more like "Empänger E-Mail" than "An E-Mail". That sounds strange. Do you use it like that in normal every-day speech?

Reply:
The question is the context this is used in. If it is ment like "recipient", then I'm with you that should translate to "Empfänger". But I thought is is ment as in the setence "should be sent **to**", then "**An**" fits better here.
Maybe @knadh could shed some light in here?
TIA

Reply:
![image](https://user-images.githubusercontent.com/547147/178977847-9db8c294-d96b-4c94-8f6e-b212b5578be0.png)

This is how it looks on the UI. Sending test message `from email` -> `to e-mail`.

Reply:
Okay, then @an0nfunc is right, I'll fix that.